contracts/peripheral/AuraViewHelpers.sol

```diff
@@ -0,0 +1,441 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.11;
+
+import { IBalancerPool, IBalancerVault } from "../interfaces/balancer/IBalancerCore.sol";
+import { AuraLocker } from "../core/AuraLocker.sol";
+import { IBooster } from "../interfaces/IBooster.sol";
+
+/**
+ * @title   AuraViewHelpers
+ * @author  AuraFinance
+ * @notice  View-only contract to combine calls
+ */
+contract AuraViewHelpers {
```

Comment:
if there are any known issue of using this views please add a note , for example on balancer https://docs.balancer.fi/reference/contracts/query-functions.html#overview
```Don't use onchain queries````